chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/UpdateMacraMeasuresJob.java

```diff
@@ -0,0 +1,68 @@
+package gov.healthit.chpl.scheduler.job;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.ff4j.FF4j;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.support.TransactionCallbackWithoutResult;
+import org.springframework.transaction.support.TransactionTemplate;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import gov.healthit.chpl.FeatureList;
+import gov.healthit.chpl.dao.MacraMeasureDAO;
+import gov.healthit.chpl.manager.CertifiedProductDetailsManager;
+
+public class UpdateMacraMeasuresJob extends QuartzJob {
+    private static final Logger LOGGER = LogManager.getLogger("updateMacraMeasuresJobLogger");
+
+    private static final String SPACER = "*********";
+    private static final String JOB_NAME = "Update Macra Measures";
+
+    private static final String OLD_G1G2_SUBSTRING = "ACI";
+    private static final String NEW_G1G2_SUBSTRING = "PI";
+
+    @Autowired
+    private MacraMeasureDAO macraMeasureDAO;
+
+    @Autowired
+    private CertifiedProductDetailsManager certifiedProductDetailsManager;
+
+    @Autowired
+    private JpaTransactionManager txnMgr;
+
+    @Autowired
+    private FF4j ff4j;
+
+    @Override
+    public void execute(final JobExecutionContext jobContext) throws JobExecutionException {
+        SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
+        if (!ff4j.check(FeatureList.EFFECTIVE_RULE_DATE)) {
```

Comment:
I think I did this in one of the jobs I wrote and then somebody said not to but I can't really remember. Most of the Cures jobs don't have this check so it might be better to leave it out for consistency. I don't have a strong feeling either way beyond the consistency argument.